Web/html3/DragAndDrop.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
```

Comment:
Try to create an External CSS, so that same CSS will use in multiple html files.